src/brogue/Rogue.h

```diff
@@ -33,7 +33,7 @@
 #define USE_UNICODE
 
 // Brogue version: what the user sees in the menu and title
-#define BROGUE_VERSION_STRING "CE 1.9.3"
+#define BROGUE_VERSION_STRING "CE 1.9.3"BROGUE_UNSTABLE_VERSION_SUFFIX
```

Comment:
Will do.

Reply:
Ah.  We're relying on implicit string literal concatenation already [here][0], which the parens break.  I could try rejiggering the help text, possibly by just dropping the reference to `BROGUE_VERSION_STRING`.  Alternately if we didn't want to mess with that we could still add the space (but not the parens.)

Or we could wrap the `#define` value here in an explicitly stringifying macro call.  That would have the same effect (the result of the macro expansion would I think literally be just this implicit string concatenation) but it would at least group the two tokens together visually.

[0]: https://github.com/tmewett/BrogueCE/blob/eb23397530ce3f0e005671930b122d24b05caa56/src/platform/main.c#L19

Reply:
Hmm ok, yeah just add the space :+1: I think literal concatenation happens very early in parsing so there are no precedence issues

Reply:
Ha, I already tweaked the help.  I'll tweak it back.